pkg/export/template.go

```diff
@@ -117,6 +146,25 @@ func templateJoin(sep string, input []interface{}) (string, error) {
 	return strings.Join(results, sep), nil
 }
 
+func (t *Template) row(fields ...interface{}) (string, error) {
+	for _, e := range fields {
+		s, err := jsonScalarToString(e)
+		if err != nil {
+			return "", fmt.Errorf("failed to write row: %v", err)
+		}
+		t.tablePrinter.AddField(s, nil, nil)
+	}
+	t.tablePrinter.EndRow()
+	return "", nil
+}
+
+func (t *Template) End() error {
+	if err := t.tablePrinter.Render(); err != nil {
```

Comment:
If there was no `row` helper used in this template, what does this do?

Reply:
After some refactoring to lazy-init the `TablePrinter`, this only gets called if `{{row}}` was used - automatically, or when `{{endtable}}` is used - whichever comes first.